js/query-parser.js

```diff
@@ -0,0 +1,118 @@
+/**
```

Comment:
indentation is 2 spaces

Reply:
For some reason all new files that I create are formatted by tabs, even though the IDE shows that it's spaces.. I need to change it to tabs, and then change it back to spaces to work.. Weird